lib/iris/analysis/_interpolation.py

```diff
@@ -142,6 +143,11 @@ def get_xy_coords(cube, dim_coords=False):
     * cube:
         An instance of :class:`iris.cube.Cube`.
 
+    Kwargs:
+
+    * dim_coords:
+        Defaults to False.
```

Comment:
Perhaps this argument should have the same description as the `dim_coords` argument to `Cube.coords()`